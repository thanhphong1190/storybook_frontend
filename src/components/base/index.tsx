export { Button, ButtonLink } from './Button'
export { SVGIcon, IconEnums } from './icon/svgIcon'
export { Icon, Icons } from './Icon'
