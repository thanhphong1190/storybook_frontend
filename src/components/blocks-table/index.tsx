export { BlocksTable, BlocksTableProps } from './BlocksTable'
