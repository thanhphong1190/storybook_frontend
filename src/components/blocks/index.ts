export { ContentBlock, ContentBlocksProps } from './ContentBlock'
export { AuthorBlock, AuthorBlockProps } from './AuthorBlock'
export { VideoBlock, PlayButton, VideoBlocksProps, VideoPlayer, VideoPlayerProps } from './VideoBlock'