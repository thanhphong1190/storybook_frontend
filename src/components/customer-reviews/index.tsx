export { default as ReviewCard } from './ReviewCard'
export { default as CustomerReviews } from './CustomerReviews'
