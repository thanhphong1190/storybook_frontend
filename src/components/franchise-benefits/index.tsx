export { FranchiseBenefits, FranchiseBenefitsProps } from './FranchiseBenefits'
export {
  FranchiseBenefitCard,
  FranchiseBenefitCardProps
} from './FranchiseBenefitCard'
