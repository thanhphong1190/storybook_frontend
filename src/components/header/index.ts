export { Header, HeaderProps } from './Header'
export { SubHeader, SubHeaderProps } from './SubHeader'
