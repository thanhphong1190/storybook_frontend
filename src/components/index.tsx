export { ContentBlock } from './blocks/ContentBlock'
export { Header } from './header/Header'
export { Navigation } from './navigation/Navigation'
export { HeroImage } from './hero/HeroImage'
export { KeyMessage } from './key-message/KeyMessage'
export { Footer } from './footer/Footer'
export { default as Banner } from './banner/Banner'
export { default as OrangeBanner } from './orange-banner/OrangeBanner'
export { ReviewCard, CustomerReviews } from './customer-reviews'
export { StandardContent } from './standard-content'
export { FranchiseBenefitCard, FranchiseBenefits } from './franchise-benefits'
export { Hero } from './hero/Hero'
export { ReadyStartedBlocks } from './ready-started-blocks'
export { BlocksTable } from './blocks-table'
