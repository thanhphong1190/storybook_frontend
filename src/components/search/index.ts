export { LocationSearchForm, LocationSearchFormProps } from './LocationSearchForm'
export { NewsSearchForm, NewsSearchFormProps } from './NewsSearchForm'