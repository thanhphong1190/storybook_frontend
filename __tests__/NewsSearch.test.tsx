import { NewsSearchForm, NewsSearchFormProps } from '../src/components/search'
import renderer from 'react-test-renderer'

const props: NewsSearchFormProps = {
  searchButtonText: 'Search',
  searchPlaceHolder: 'Search Placeholder',
  displaySearch: true,
  onSearchClose: () => {},
  onSearchClick: () => {}
}
it('renders correctly', () => {
  const tree = renderer.create(<NewsSearchForm {...props} />).toJSON()
  expect(tree).toMatchSnapshot()
})
