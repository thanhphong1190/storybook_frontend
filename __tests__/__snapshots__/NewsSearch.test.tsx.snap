// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="SearchFormstyled__NewsSearchWrapper-sc-1lsr3ed-4 eHxgzd"
>
  <label
    className="SearchFormstyled__LabelIcon-sc-1lsr3ed-6 fbkwdf"
  >
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-search fa-w-16 "
      data-icon="search"
      data-prefix="fas"
      focusable="false"
      role="img"
      style={Object {}}
      viewBox="0 0 512 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M505 442.7L405.3 343c-4.5-4.5-10.6-7-17-7H372c27.6-35.3 44-79.7 44-128C416 93.1 322.9 0 208 0S0 93.1 0 208s93.1 208 208 208c48.3 0 92.7-16.4 128-44v16.3c0 6.4 2.5 12.5 7 17l99.7 99.7c9.4 9.4 24.6 9.4 33.9 0l28.3-28.3c9.4-9.4 9.4-24.6.1-34zM208 336c-70.7 0-128-57.2-128-128 0-70.7 57.2-128 128-128 70.7 0 128 57.2 128 128 0 70.7-57.2 128-128 128z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
  </label>
  <div
    className="SearchFormstyled__SearchFormGroup-sc-1lsr3ed-3 SearchFormstyled__NewsSearchGroup-sc-1lsr3ed-5 fqjwpL"
  >
    <input
      className="SearchFormstyled__NewsSearchTextInput-sc-1lsr3ed-7 gQWCga"
      placeholder="Search Placeholder"
    />
    <button
      className="Button-lxhj0a-0 caAbrL"
      color="white"
      onClick={[Function]}
    >
      Search
    </button>
    <a
      className="Button__ButtonLink-lxhj0a-1 gEZLzW"
      onClick={[Function]}
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-times fa-w-11 "
        data-icon="times"
        data-prefix="fas"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 352 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
    </a>
  </div>
</div>
`;
